drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -1065,14 +1065,10 @@ Ref<Image> RasterizerSceneGLES3::sky_bake_panorama(RID p_sky, float p_energy, bo
 	texture.target = GL_TEXTURE_2D;
 	texture.active = true;
 	texture.tex_id = rad_tex;
-	texture.is_render_target = true;
+	texture.is_render_target = false;
 
 	GLES3::TextureStorage::get_singleton()->texture_2d_initialize_from_texture(tex_rid, texture);
 	Ref<Image> img = GLES3::TextureStorage::get_singleton()->texture_2d_get(tex_rid);
-	GLES3::Utilities::get_singleton()->texture_free_data(rad_tex);
-
-	texture.is_render_target = false;
-	texture.tex_id = 0;
```

Comment:
These lines don't do anything, and don't modify the texture in `texture_storage`.